build/charts/antrea/values.yaml

```diff
@@ -10,6 +10,8 @@ trafficEncapMode: "encap"
 # -- Tunnel protocol used for encapsulating traffic across Nodes. It must be one
 # of "geneve", "vxlan", "gre", "stt".
 tunnelType: "geneve"
+# -- TunnelPort is the port for the TrafficEncapMode Tunnel.
+tunnelPort:
```

Comment:
can it work if just leaving it unset? I'm not sure if it needs to use 0 to represent empty value and let code to select proper tunnel port when it's empty value.

Reply:
Thanks for the review, updated.